Automattic/wp-calypso
# Gutenboarding: Remove FSE starter designs, replace with Seedlet Blocks

Fixes https://github.com/Automattic/wp-calypso/issues/46537

#### Changes proposed in this Pull Request

* Remove older FSE themes and add Seedlet blocks when selecting a theme.

![96298246-c1377100-0fb7-11eb-8762-6c0c380a731e](https://user-images.githubusercontent.com/1464705/97360834-f1a2c900-185b-11eb-9d55-513b5c9cfd88.png)
 
#### Testing instructions

* Apply this PR locally
* Sandbox public-api.wordpress.com
* Apply D51883-code diff
* Boot calypso locally.
* Go to http://calypso.localhost:3000/new/design?flags=gutenboarding/site-editor
* Confirm you only see "Seedlet" as a design option.
* Create a site and confirm you can edit seedlet using the site editor.
